app/src/main/java/com/monkeyteam/chimpagne/ui/FindEventScreen.kt

```diff
@@ -118,6 +118,7 @@ fun MainFindEventScreen(
     accountViewModel: AccountViewModel
 ) {
 
+  val accountUIState by accountViewModel.uiState.collectAsState()
```

Comment:
Should be deleted if not used

Reply:
Indeed, you understand my little memory lapse.